test/images/webhook/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:latest
+FROM BASEIMAGE
```

Comment:
I was going to nit on BASEIMAGE making this an incomplete `Dockerfile`, in which case I'd want to rename it to `Dockerfile.in`,`Dockerfile.build` or something... but I guess this is the pattern used all throughout test/images already.  So.. carry on.

Anyone know if there's an issue for this?

Reply:
No I don't see that huge problem with it either. We use  `make build` consistently instead to put all other prerequisites in place as well, even though the Dockerfile wouldn't be a template.